src/main/java/Calculator/Calculator.java

```diff
@@ -0,0 +1,26 @@
+package Calculator;
+
+import Calculator.Interface.BaseCalculatorInterface;
+
+public class Calculator implements BaseCalculatorInterface {
+
+    public double addition(double a, double b) {
```

Comment:
We usually add javadoc to public methods. I'll show how: type _/** [Enter]_ before the method.

Reply:
Added